dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerTypeReport.java

```diff
@@ -102,22 +102,22 @@ public boolean isEmpty()
         return getErrorReports().isEmpty();
     }
 
-    public void addObjectReport( TrackerObjectReport objectReport )
+    public void addEntity( Entity objectReport )
```

Comment:
`objectReport` should then be `entity`

Why does this `Entity` have an index? Why does the position matter?

Reply:
I am not sure.
I am just renaming in this PR, I will check on these details on a later PR